scripts/release-changelog.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+prev_tag="$(source scripts/tags.sh; previous_tag)"
```

Comment:
you've taught me something about shells.

Reply:
Actually, I can't take credit for this. I just learned this a couple weeks ago :)